Extension/src/LanguageServer/ui.ts

```diff
@@ -296,6 +309,21 @@ export class OldUI implements UI {
         }
     }
 
+    private compilerTimout?: NodeJS.Timeout;
+    public async showCompilerStatusIcon(show: boolean): Promise<void> {
+        if (this.compilerTimout) {
+            clearTimeout(this.compilerTimout);
+        }
+        const action: string = "";
+        if (show) {
+            this.compilerTimout = setTimeout(() => { this.compilerStatusItem.show(); }, 15000);
+            telemetry.logLanguageServerEvent('compilerStatusBar', { 'compiler status bar shown' : action });
```

Comment:
This is missing from the newUI case?

Reply:
Also, shouldn't the telemetry be when it's actually shown?

Reply:
I am not sure what you mean by "when it's actually shown". Do you mean put it after the "show" function call?

Reply:
It means put it inside the lambda for the `setTimeout`.

Also delete the `{ 'compiler status bar shown' : action }` part. `action` should be on the left of the colon, but we don't need this anyway.

Reply:
yeah